### PR TITLE
feat: Added Image and Input components

### PR DESCRIPTION
**Pull request by arjunbharti**
Created Image and Input components
**Image:**

1. Responsive Image
2. Rounded Image

**Input:**

1. Text box
2. With validation and error style

### CODE REVIEW

File: styles/utils.css [hunk 1]
```diff
@@ -17,5 +17,87 @@
...
+    width: 80rem;
+}
+
+.h-90{
+    width: 90rem;
+}
+
+.h-100{
+    width: 100rem;
 }
```
Comment (darshandamre) [inline styles/utils.css:103]: you have given width property to height classes

File: documentation/index.html [hunk 2]
```diff
@@ -776,6 +776,114 @@ <h3 class="content-heading">Getting Started</h3>
...
+                            name=""
+                            id=""
+                            class="input input-label-warning"
+                            placeholder="Enter your email here..."
+                        >
+                        <label for="" class="input-label">Label</label>
+                        <input 
+                            type="email" 
+                            name=""
+                            id=""
```
Comment (darshandamre) [inline documentation/index.html]: you can also remove the empty attributes here

File: styles/utils.css (hunk 1 above)
Comment (arjunbharti) [reply styles/utils.css:103]: Oh no, I think I mixed few things while resolving the merge conflicts. Will fix this!

**Review by srv-pranjal — APPROVED** [review]
Other than the changes suggested, Everything looks good.

File: documentation/index.html (hunk 2 above)
Comment (arjunbharti) [reply documentation/index.html]: fixed, thanks

**Pull request merged**